scheduler/kube/kube.go

```diff
@@ -104,6 +105,24 @@ func (s *kubeScheduler) Schedule(ctx context.Context, stage *core.Stage) error {
 	rand := strings.ToLower(uniuri.NewLen(12))
 	name := fmt.Sprintf("drone-job-%d-%s", stage.ID, rand)
 
+	var mounts []v1.VolumeMount
+        mount := v1.VolumeMount{
+		Name:           "local",
+		MountPath:      filepath.Join("/tmp", "drone"),
+	}
+        mounts = append(mounts, mount)
+
+	var volumes []v1.Volume
+        source := v1.HostPathDirectoryOrCreate
+        volume := v1.Volume{
+		Name:            "local",
```

Comment:
is local volume the right storage type? we are using host volume mounts with HostPathDirectoryOrCreate. For reference https://github.com/drone/drone-runtime/blob/master/engine/kube/util.go#L137

Reply:
@bradrydzewski Is that not what I'm doing right below here in the next 5 lines?  I just named it "local", but we can name it whatever we want.  I tried to replicate the same functionality as the runtime.  I specify the source as v1.HostpathDirectoryOrCreate on 116 and use that as the source on line 122.

Reply:
sorry, I misread the code and thought it was creating a `local` mount instead of a `hostPath` mount. That part looks good. One question I have .... does `local` need to be a unique name? What will happen if two pipelines run on the same machine and try to create `local` volumes?

Reply:
Good point on the potential name collision.  I don't know if it would cause an issue or not, but I agree we should probably use a more unique or at least explicitly named volume.  The name in the runtime code is obfuscated by the engine spec struct so I wasn't able copy that.  I thought about maybe using the namespace name.  Any other suggestions?

Reply:
@bradrydzewski I've updated the volume name to be unique.  Please take a look.